cmd/kubeadm/app/util/version.go

```diff
@@ -49,7 +49,11 @@ var (
 //  latest-1.0  (and similarly 1.1, 1.2, 1.3, ...)
 func KubernetesReleaseVersion(version string) (string, error) {
 	if kubeReleaseRegex.MatchString(version) {
-		return version, nil
+		if version[0] == 'v' {
```

Comment:
first part (HasPrefix) ok, will update. for fmt.Sprintf, I think for such trivial operation, + operator will be a lot faster than Sprintf.